envoy/upstream/outlier_detection.h

```diff
@@ -82,14 +82,17 @@ using ExtResult = absl::variant<HttpCode, LocalOriginEvent>;
 // Each monitor may implement different health detection algorithm.
 class ExtMonitor {
 public:
-  using ExtMonitorCallback =
-      std::function<void(uint32_t /* enforce */, std::string /* runtime_key */)>;
   virtual ~ExtMonitor() {}
```

Comment:
nit: `virtual ~ExtMonitor() = default;` for clang tidy.

Reply:
Done. In this and several other places. Thanks!